fossunited/chapters/doctype/foss_chapter/foss_chapter.py

```diff
@@ -25,7 +25,7 @@ class FOSSChapter(WebsiteGenerator):
         chapter_lead: DF.Link | None
         chapter_members: DF.Table[FOSSChapterLeadTeamMember]
         chapter_name: DF.Data
-        chapter_type: DF.Literal["City Community", "FOSS Club", "Conference"]
+        chapter_type: DF.Literal["City Community", "FOSS Club", "Conference"]  # type: ignore
```

Comment:
it's not obvious why this type hint is being ignored.

Reply:
oof, this was not supposed to be pushed. was experimenting